lochness/box/__init__.py

```diff
@@ -438,6 +438,22 @@ def sync_module(Lochness: 'lochness.config',
                          compress=compress, delete=False,
                          dry=False)
 
+                    # check if api token is still valid after each data pull
+                    try:
+                        _ = client.user().get()
+                    except BoxOAuthException:
+                        api_token = get_access_token(client_id,
+                                                     client_secret,
+                                                     user_id)
+
+                        # box authentication
+                        auth = OAuth2(
+                            client_id=client_id,
+                            client_secret=client_secret,
+                            access_token=api_token,
+                        )
+                        client = Client(auth)
```

Comment:
I would not recommend it from my experience with Mediaflux slowness since that has to authenticate for every data pull as small as one survey json.

My counter recommendation stays the same as https://github.com/AMP-SCZ/lochness/pull/50#issuecomment-923313012 for future.

Reply:
It's not executing the authentication after every pull. It's checking first if `Client` is still connected, and if not refresh the connection. Would you still not recomment this way of flow?